games/godot/Portfile

```diff
@@ -50,7 +49,7 @@ subport ${name}-3.2 {
     PortGroup       obsolete 1.0
 
     version         3.2.3
-    revision        2
+    revision        1
```

Comment:
`revision` should not decrement if the version number is not changing.

Reply:
That was a mistake.... it should still be 2. I will go and fix that.

Reply:
Okay, it should be fixed now. Nice catch.